include/CAN/can_mapping.h

```diff
@@ -31,7 +31,7 @@ CPP_GUARD_BEGIN
  * match the can message based on the specified CAN mapping ID and ID mask
  * @param can_msg the CAN message to test
  * @param mapping the can mapping to check against
- * @return true if the CAN message matches the mapping's CAN ID and ID mask
+ * @return true if the CAN message matches the mapping's CAN ID and ID mask, or true if mapping's CAN ID is 0
```

Comment:
Why 0?  What is special about that?  Does this not break behavior of the function?